x-pack/test/security_solution_cypress/cypress/tasks/navigation.ts

```diff
@@ -91,6 +92,12 @@ export const visitUserDetailsPage = (userName = 'test') => {
   visitWithTimeRange(userDetailsUrl(userName));
 };
 
+export function goBackToRulesTableViaBreadcrumbs(): void {
```

Comment:
It's not necessarily "back", it could be "forward" as well.

```suggestion
export function openRuleManagementPageViaBreadcrumbs(): void {
```